server/src/test/java/com/linecorp/centraldogma/server/internal/api/converter/WatchRequestConverterTest.java

```diff
@@ -51,6 +51,36 @@ void convertWatchRequest() throws Exception {
         assertThat(watchRequest.timeoutMillis()).isEqualTo(10000); // 10 seconds
     }
 
+    @Test
+    void convertRequestWithETaggedIfNoneMatchHeaderToWatchRequest() throws Exception {
+        final ServiceRequestContext ctx = mock(ServiceRequestContext.class);
+        final AggregatedHttpRequest request = mock(AggregatedHttpRequest.class);
+        final RequestHeaders headers = RequestHeaders.of(HttpMethod.GET, "/",
+                                                         HttpHeaderNames.IF_NONE_MATCH, "\"-1\"",
+                                                         HttpHeaderNames.PREFER, "wait=10");
+        when(request.headers()).thenReturn(headers);
+
+        final WatchRequest watchRequest = convert(ctx, request);
+        assertThat(watchRequest).isNotNull();
+        assertThat(watchRequest.lastKnownRevision()).isEqualTo(Revision.HEAD);
+        assertThat(watchRequest.timeoutMillis()).isEqualTo(10000); // 10 seconds
+    }
+
+    @Test
+    void convertRequestWithWeakComparisonETaggedIfNoneMatchHeaderToWatchRequest() throws Exception {
```

Comment:
How about just making valid and invalid cases and renaming those to `validIfNoneMatch` and `invalidIfNoneMatch`?